assets/js/modules/analytics/components/settings/SettingsEdit.test.js

```diff
@@ -20,18 +20,26 @@
  * Internal dependencies
  */
 import { render, waitFor, createTestRegistry } from '../../../../../../tests/js/test-utils';
-import { STORE_NAME } from '../../datastore/constants';
 import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import { STORE_NAME } from '../../datastore/constants';
+import { MODULES_ANALYTICS_4 } from '../../../analytics-4/datastore/constants';
 import SettingsEdit from './SettingsEdit';
 import * as fixtures from '../../datastore/__fixtures__';
-import { provideSiteInfo } from '../../../../../../tests/js/utils';
+import { provideModules, provideSiteInfo } from '../../../../../../tests/js/utils';
 
 describe( 'SettingsEdit', () => {
 	let registry;
 
 	beforeEach( () => {
 		registry = createTestRegistry();
 		provideSiteInfo( registry );
+		provideModules( registry, [
+			{
+				slug: 'analytics',
+				activate: true,
```

Comment:
This should be `active` instead.
```suggestion
				active: true,
```